test/helper.js

```diff
@@ -22,8 +22,8 @@ const resetDatabase = async () => {
   // clear all test data (order matters due to foreign key relationships)
   await models.sequelize.query(`
     DELETE FROM "Users";
-    DELETE FROM "Categories";
     DELETE FROM "Resources";
+    DELETE FROM "Categories";
```

Comment:
I had to change the order of deletion for the test helper reset to work. Because of REFERENTIAL INTEGRITY, you cannot delete Categories that are being used/referenced by Resources. Only after all the Resources are deleted, can you then delete the Categories.